biome.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "vcs": {
+    "enabled": true,
+    "clientKind": "git",
+    "useIgnoreFile": true,
+    "defaultBranch": "main"
+  },
+  "files": {
+    "ignoreUnknown": true,
+    "ignore": ["submit.astro"]
```

Comment:
Can we use an ignore comment in the file instead?

Reply:
sure, I copied this from an template @alexanderniebuhr provided me, any specific files you want to ignore?

Reply:
Yeah this is my bad, the example file, had custom logic for that repo. We don't need the ignore in this repo.